src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,190 @@
+package subway;
+
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.annotation.DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD;
+import static subway.SubwayFixture.LINE_SHIN_BUN_DANG_REQUEST;
+import static subway.SubwayFixture.LINE_TWO_REQUEST;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.jdbc.Sql;
+
+
+@DisplayName("지하철 노선 관련 기능")
+@DirtiesContext(classMode = BEFORE_EACH_TEST_METHOD)
+@Sql("/stations.sql")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철노선을 생성한다.")
+    @Test
+    void 지하철_노선_생성() {
+
+        //when
+        ExtractableResponse<Response> response = createSubwayLine(LINE_SHIN_BUN_DANG_REQUEST);
+
+        //then
+        assertThat(response.jsonPath().getString("name")).contains("신분당선");
+        List<String> lineNames = getSubwayLines().jsonPath().getList("name", String.class);
+        assertThat(lineNames).containsAnyOf(LINE_SHIN_BUN_DANG_REQUEST.getName());
```

Comment:
```suggestion
        //when
        createSubwayLine(LINE_SHIN_BUN_DANG_REQUEST);

        //then
        List<String> lineNames = getSubwayLines();
        assertThat(lineNames).containsAnyOf(LINE_SHIN_BUN_DANG_REQUEST.getName());
```

한번 더 리팩토링을 하자면, 이렇게도 표현이 가능할 것 같아요! 

더 명확하게 보여주기 위해서라고 생각해주시면 되겠습니다! 🤩

Reply:
👍